changelog/unreleased/rework-virus-handling.md

```diff
@@ -0,0 +1,5 @@
+Enhancement: Improve infected file handling
```

Comment:
```suggestion
Bugfix: Improve infected file handling
```

Reply:
This was in the scope of a bugfix. Virus files were scanned over and over again, so from a consumer POV it was a Bug.